src/ucp/core/ucp_request.inl

```diff
@@ -852,33 +852,24 @@ ucp_request_get_memory_type(ucp_context_h context, const void *address,
 }
 
 static UCS_F_ALWAYS_INLINE void
-ucp_ep_ptr_map_check_status(ucp_ep_h ep, void *ptr, const char *action_str,
-                            ucs_status_t status)
+ucp_request_ptr_map_check_status(ucp_ep_h ep, void *ptr, const char *action_str,
+                                 ucs_status_t status)
 {
     ucs_assertv((status == UCS_OK) || (status == UCS_ERR_NO_PROGRESS),
                 "ep %p: failed to %s id for %p: %s", ep, action_str, ptr,
                 ucs_status_string(status));
 }
 
-static UCS_F_ALWAYS_INLINE ucs_status_t
-ucp_ep_ptr_id_alloc(ucp_ep_h ep, void *ptr, ucs_ptr_map_key_t *ptr_id_p)
-{
-    ucs_status_t status;
-
-    status = UCS_PTR_MAP_PUT(request, &ep->worker->request_map, ptr,
-                             ucp_ep_use_indirect_id(ep), ptr_id_p);
-    ucp_ep_ptr_map_check_status(ep, ptr, "allocate", status);
-
-    return status;
-}
-
 static UCS_F_ALWAYS_INLINE void ucp_send_request_id_alloc(ucp_request_t *req)
 {
     ucp_ep_h ep = req->send.ep;
     ucs_status_t status;
 
     ucp_request_id_check(req, ==, UCS_PTR_MAP_KEY_INVALID);
-    status = ucp_ep_ptr_id_alloc(ep, req, &req->id);
+    status = UCS_PTR_MAP_PUT(request, &ep->worker->request_map, req,
+                             ucp_ep_use_indirect_id(ep), &req->id);
+    ucp_request_ptr_map_check_status(ep, req, "allocate", status);
```

Comment:
```suggestion
    ucp_request_ptr_map_status_check(status, "PUT", ep, req);
```
or even maybe add a helper macro like `UCP_REQUEST_PTRMAP_PUT(&ep->worker->request_map, req)` with put+check inside?

Reply:
Applied suggested changes.

Macro looks redundant for me, cause it doesn't simplify the code too much to be introduced, IMO.